AmericanMedicalAssociation/joe-style-guide
# EWLJ-876: Only One Video/Audio should Play in Horizontal Gallery Component

## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**

- [EWLJ-876: Only One Video/Audio should Play in Horizontal Gallery Component](https://github.com/AmericanMedicalAssociation/code-of-ethics/pull/394)

**Jira Ticket**

- [EWLJ-876: Only One Video/Audio should Play in Horizontal Gallery Component](https://ama-it.atlassian.net/browse/EWLJ-876).


## Description



## To Test

1. on main repo pull the branch EWLJ-876 **https://github.com/AmericanMedicalAssociation/code-of-ethics/pull/394**
2. styleguide/joe-styleguide lando gulp serve
3. lando drush @journalofethics.local cr
4. Make sure you have audio content on the fields 
5. go to page /exhibition/portraits-care 

## Visual Regressions



## Relevant Screenshots/GIFs


https://github.com/user-attachments/assets/67560dfc-0bc3-4a72-b260-493b408f86b9



## Remaining Tasks

When this is approved create the new tag style.

## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
https://github.com/AmericanMedicalAssociation/code-of-ethics/pull/394
